### PR TITLE
Deferred ring and 5 hour requirement

### PR DESCRIPTION
**Pull request by gacarini**
Removing the callout about the reports not being enabled for the deferred ring and adding the 5 hour minimum requirement to the limitations.

### CODE REVIEW

File: SharePoint/SharePointOnline/sync-health.md [hunk 1]
```diff
@@ -229,6 +229,8 @@ This section describes known limitations and considerations in sync reporting.
 
 **Sync app version: Mac App Store edition** For devices using the Mac App Store edition of the sync app, the version installed on each device is displayed in the **Devices** tab. The dashboard doesn't currently track whether or not the Mac App Store edition is the latest version of the sync app available in the Mac App Store. If any devices use this edition, they'll be excluded from the **Sync app version** section of the **Overview** tab and the number of excluded devices is displayed. This is the expected result.
 
+**Minimum time device is on for eligibility:** Devices need to be turned on for a minimum of 5 hours to be eligible for the report. Devices that are turned off frequently and not on for that amount of time, may be missing from the dashboard, even if the group policy is set.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sync-health.md:232]:
```suggestion
**Minimum time device is on for eligibility:** Devices need to be turned on for a minimum of five hours to be eligible for the report. Devices that are turned off frequently and not on for that amount of time might be missing from the dashboard, even if the group policy is set.
```

File: SharePoint/SharePointOnline/sync-health.md
Comment (libbygentry) [inline SharePoint/SharePointOnline/sync-health.md:1]: Thanks for these changes, Gaia! Can you also remove step 1 from the Troubleshooting section? It's no longer relevant with deferred ring now being available. That would also mean step "2" likely doesn't need to be numbered any longer (but that check itself should definitely remain in the section)

Comment (libbygentry) [inline SharePoint/SharePointOnline/sync-health.md:1]: "If the setting was applied correctly, wait for 36 hours..." to remove the mention of Insiders or Production on that line too (talking about bottom of Troubleshooting section

Comment (gacarini) [reply SharePoint/SharePointOnline/sync-health.md:1]: Ok sounds good!

**Comment by gacarini** [comment]
@JohanFreelancer9 - I've addressed all the comments but it seems like now I can't merge the pull request. Any suggestions?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 - I've addressed all the comments but it seems like now I can't merge the pull request. Any suggestions?

@scanum, can you please assist? Thanks.

**Comment by gacarini** [comment]
@scanum  - It says only folks with write access can merge pull requests. Is that something you can help with?

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**